scripts/APPS-162-delete-emails.js

```diff
@@ -119,7 +119,7 @@ async function clearFromLogs(topicIds) {
 	
 	let logCollectionsToClearBySubsCodes = ["notify_tooManyReq_logs", "notify_badEmail_logs"];
 	
-	for (coll in logCollectionsToClearBySubsCodes) {
+	for (let coll in logCollectionsToClearBySubsCodes) {
```

Comment:
This will not fix the issue. May be you can use a forEach statement?

```suggestion
	logCollectionsToClearBySubsCodes.forEach( coll ) {
	
	...
	
	} );
```

Reply:
Really?
I use "for (let" all the time in this script.  Should I fix em all?

Reply:
Ah!  Got it!  Should be:
for (let coll of 
OF!  Not in!
That's hwhat i use elsewhere
and it works!

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of